### DISCUSSION TITLE
How to best configure network (Unifi + Gl.iNet + Cellular + Cable)

### DISCUSSION THREAD

**Discussion by chansearrington** [SmoothWAN/SmoothWAN#82] [General & Help]
I got my v1.0 SmoothWAN network setup quickly using these guides and the other discussion topics (THANKS!)

My GL.iNet Slate device gets here today and I'd like to make some changes, but I'm not sure how to best configure all of this. I'm not sure I can get it do what I'd like it to do and I don't know what to ask/search for with the right lingo, lol.

Short version... in order to get my v1 setup, I had to treat my SmoothWAN Raspberry PI as the router and DHCP server on my network.

But doing this means I can't manage my devices and network through Unifi Console.

So how do I treat the SmoothWAN device as a modem. BUT... still do things like VPN Policy Routing to keep my servers off of Speedify and reroute certain web traffic off speedify.

What is this called? What am I trying to do? I'm happy to search and try to figure it out, but I don't know what I'm searching for.

If this helps, I made a little network diagram example.

![CleanShot 2022-12-23 at 13 31 18](https://user-images.githubusercontent.com/123082/209399116-21fbe0b4-df3c-4b8f-a325-b4af2e18a3e1.png)

**Comment by TalalMash** [comment]
Hello!
SmoothWAN as a modem sounds similar to "drop-in gateway" mode, which is found in many Multi-WAN appliances, I believe GL.iNet stock firmware includes it too, but they often use arp-spoofing which often introduce issues not to mention latency and bandwidth.

The network diagram is pretty simple and does not need drop-in mode, I'm not very familiar with the Unifi but have you tried connecting the LAN of the Slate/Pi to the Unifi LAN, then disable DHCP in SmoothWAN and enable in the Unifi? PBR will still work.
You will then have the Unifi as DHCP server (not routing), Slate as gateway/router, or do certain features require routing on the Unifi side? (e.g statistics, QoS, firewall etc)

**Reply by TalalMash** [reply]
I believe UDM includes PBR as well, you could use VLAN for each WAN and let UDM manage multiple routing tables and rules.

**Reply by chansearrington** [reply]
I'd like to have the UDM as the router.

How do I configure the VLANs for the WANs on the SmoothWAN side that the UDM will be able to see?

**Reply by TalalMash** [reply]
https://smoothwan.com/tips/ VLAN section
